tools/hls-verifier/src/Utilities.cpp

```diff
@@ -111,8 +121,9 @@ bool compareFiles(const string &refFile, const string &outFile,
                             "] are not equal.");
       return false;
     }
-    if (str1 == "[[[/runtime]]]")
+    if (str1 == "[[[/runtime]]]") {
```

Comment:
no braces here

Reply:
I added them here because all other ifs in the chain use braces. According to the guidelines "... readability is also harmed if an if/else chain does not use braced bodies for either all or none of its members ..."

Reply:
I always understood this rule as applying to a chain of if/else if/.../else if/else and not adjacent ifs as is the case here, but whatever, fine either way.